### PR TITLE
Fix last day of session excluded from current session and balance that doesn't update

### LINKED ISSUE #97: The balance doesn't update
**Describe the bug**
I paid a part of my balance and the profile on the app doesn't display the new balance (but beta does). It's like the refresh doesn't happen. 

**To Reproduce**
1. Go to profile (first time to create the cache)
2. Kill the application
3. Pay a part of your balance then wait 2-3 days
4. Reopen the application and go to the profile section.

**Expected behavior**
The profile should update itself each time the profile is loaded.

### PR DESCRIPTION
**Pull request by apomalyn**
It's been some time since I did a PR ahah.

This PR contains 2 fixes:
- the last day of the session was excluded from the active session which prevented the schedule from being updated and prevented the progress bar to reach 100%. 
- fix #97 :)

### CODE REVIEW

File: lib/core/managers/user_repository.dart [hunk 1]
```diff
@@ -260,7 +260,7 @@ class UserRepository {
       _logger.d("$tag - getInfo: $fetchedInfo info fetched.");
 
       if (_info != fetchedInfo) {
-        _info ??= fetchedInfo;
+        _info = fetchedInfo;
```
Comment (MysticFragilist) [inline lib/core/managers/user_repository.dart:263]: Niice fix I would have never guess hahaha! 😆 What does `??=` even means? Is it like only assign when not null?

**Review by MysticFragilist — DISMISSED** [review]
LGTM! 🍾

File: lib/core/managers/user_repository.dart (hunk 1 above)
Comment (apomalyn) [reply lib/core/managers/user_repository.dart:263]: The debugger helped me a lot :laughing: 
So `a ?? b` will return `b` if `a` is null BUT `a ??= b` will assign `b` to `a` ONLY if `a` is null...that was the issue, as long as there was a cache the info will not update.

**Comment by apomalyn** [comment]
This PR contains also a fix for the `MissingPluginException`. The problem was that when you run all the tests, they are executed in parallel and so the tearDown of some test happens after the setup of another. This isn't a problem for the managers, etc. because they have their own instances BUT FlutterToast is global...

**Comment by apomalyn** [comment]
@MysticFragilist @camillebrulotte @AlexandreMarkus @JonathanDuvalV if you can review this PR ASAP that would be perfect :)

**Review by AlexandreMarkus — DISMISSED** [review]
LGTM!

**Comment by AlexandreMarkus** [comment]
@apomalyn On my phone it says "There was a problem while parsing the package" and I can't install it

**Comment by apomalyn** [comment]
@AlexandreMarkus my bad I mess up the change on the CI XD fixing that now

**Review by MysticFragilist — APPROVED** [review]
LGTM!

**Comment by apomalyn** [comment]
@MysticFragilist Sorry you will have to check that again, after the pipeline passed XD

**Comment by MysticFragilist** [comment]
No problem!

**Review by AlexandreMarkus — APPROVED** [review]
LGTM!

**Review by camillebrulotte — APPROVED** [review]
LGTM :D

**Pull request merged**